modules/gitops-installing-argocd-cli-on-linux.adoc

```diff
@@ -0,0 +1,76 @@
+// Module is included in the following assemblies:
+//
+// * installing_gitops/installing-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-cli-on-linux"]
+= Installing the {gitops-title} CLI on Linux
+
+For Linux distributions, you can download the {gitops-shortname} `argocd` CLI as a `tar.gz` archive.
```

Comment:
The ".gz" portion means that this is a compressed archive, not just a ".tar" archive. :slightly_smiling_face: